<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QMainWindow" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>loginwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/images/bg_login2.jpg&quot;);
background-position: center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="texto_frase">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,79,0);
font: 9pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>&quot;Sempre imaginei que o paraíso fosse uma
espécie de biblioteca.&quot;   - Jorge Luis Borges</string>
    </property>
   </widget>
   <widget class="QLabel" name="texto_creditos">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,79,0);
font: 6pt &quot;Times New Roman&quot;;
</string>
    </property>
    <property name="text">
     <string>Por: Rafaela Savaris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sair">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Times New Roman&quot;;
background: rgb(255, 82, 223);</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="texto_bem_vindo">
       <property name="styleSheet">
        <string notr="true">color: rgb(28, 79, 0);
font: 20pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Bem-vindo! Faça login</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="texto_username">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 79, 0);
font: 12pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="insert_username">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="texto_senha">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 79, 0);
font: 12pt &quot;Times New Roman&quot;;</string>
         </property>
         <property name="text">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="insert_senha">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_login">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;
background: rgb(0, 79, 0);</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_sign_up">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background: rgb(98, 176, 70);
font: 8pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Não tem uma conta? Cadastre-se!</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="texto_erro">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 10pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>texto_frase</zorder>
   <zorder>texto_creditos</zorder>
   <zorder>btn_sair</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
   </widget>
   <addaction name="menuLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
